src/app/plan/components/Wizard/ResourceSelectForm.tsx

```diff
@@ -88,6 +87,7 @@ const ResourceSelectForm = props => {
     const matchingCluster = clusterList.find(c => c.MigCluster.metadata.name === value);
     if (matchingCluster) {
       setFieldValue('sourceCluster', value);
+      setFieldValue('selectedNamespaces', []);
```

Comment:
Should seperate this into its own PR for testing purposes

Reply:
@ibolton336, right, that's a mistake, thanks.